titus-common-runtime/src/test/java/com/netflix/titus/runtime/clustermembership/service/ClusterMembershipConnectorStub.java

```diff
@@ -142,4 +142,10 @@ void becomeLeader() {
                 )
                 .build();
     }
+
+    private void emitEvent(ClusterMembershipEvent event) {
+        synchronized (eventProcessor) {
```

Comment:
it might be better to use a `DirectProcessor#sink()` to send events to the processor, which serializes all `onNext` events.

Reply:
Good idea. I did not know about that one.